prime-router/src/main/kotlin/Report.kt

```diff
@@ -834,9 +843,26 @@ class Report {
                     header.reportFile.reportId,
                     header.reportFile.schemaName,
                     header.receiver?.format ?: error("Internal Error: ${header.receiver?.name} does not have a format"),
-                    header.reportFile.createdAt
+                    header.reportFile.createdAt,
+                    metadata = Metadata.provideMetadata()
                 )
             }
         }
+
+        private fun Row.getStringOrNull(columnName: String): String? {
+            return try {
+                this.getString(columnName)
+            } catch (_: Exception) {
+                null
+            }
+        }
+
+        private fun Row.getStringOrDefault(columnName: String, default: String? = null): String? {
+            return try {
+                this.getString(columnName)
+            } catch (_: Exception) {
+                default
+            }
+        }
```

Comment:
Added two extension methods to let me safely get the data if it exists in the report otherwise ignore the problem.